src/MSALWrapper/TokenResult.cs

```diff
@@ -32,6 +32,22 @@ public enum AuthType
         IntegratedWindowsAuthenticationFlow,
     }
 
+    /// <summary>
+    /// Auth type extensions.
+    /// </summary>
+    public static class AuthTypeExtensions
+    {
+        /// <summary>
+        /// Checks if authType is silent.
+        /// </summary>
+        /// <param name="authType">The auth type.</param>
+        /// <returns>The <see cref="bool"/>.</returns>
+        public static bool IsSilent(this AuthType authType)
+        {
+            return authType == AuthType.Silent;
+        }
+    }
+

```

Comment:
My other main comment is about this - we should not need this, and instead can better encode the concept of interaction by using better types.